docs/desktop-wallet/support/troubleshooting.md.blade.php.md

```diff
@@ -0,0 +1,344 @@
+</details>
+{{--  --}}
+
+{{-- 7) is all other Ledger software closed? --}}
+<details><summary><u><b>7) is all other Ledger software closed?</b></u></summary>
+A user should check that <b>all</b> other <b>software</b> or <b>apps</b> that <b><i>can</i> connect</b> to a <b>Ledger</b> device are <b>closed</b>.
+
+This includes other <b>wallets</b>, <b>Ledger Live</b>, as well as any <b>browser apps</b> like <b>MetaMask</b> or <b>MEW</b>.
+
+This is because a <b>Ledger</b> device only allows <b>one</b> connection at a time.
+</details>
+{{--  --}}
+
+{{-- 8) is Firewall or Antivirus software blocking the connection? --}}
+<details><summary><u><b>8) is Firewall <i>or</i> Antivirus software blocking the connection?</b></u></summary>
+Users should also check that <i><b>no</b></i> <b>Firewall</b> or <b>Antivirus</b> software is <b>blocking</b> the <b>USB connection</b>.
+</details>
+{{--  --}}
+
+---
+
+If the **Ledger** issue is related to <u>**signing a transaction**</u>, there are a few things a user can check.
+
+{{-- 1) is the ARK Desktop Wallet up-to-date? --}}
+<details><summary><u><b>1) is the ARK Desktop Wallet up-to-date?</b></u></summary>
+A user should check that the <b>most recent</b> version of the <b>ARK Desktop Wallet</b> is <b>installed</b>.
+
+All official releases can be found by visiting the following page:
+
+<livewire:embed-link url="https://ark.io/download#desktop-wallet" caption="ARK.io Downloads | Download Blockchain Software" />
+</details>
+{{--  --}}
+
+{{-- 2) is the ARK App up-to-date? --}}
+<details><summary><u><b>2) is the ARK App up-to-date?</b></u></summary>
+A user should ensure that the <b>ARK App</b> is <b>up-to-date</b>, with the <b>most recent</b> version installed.
+
+For guidance on <i><b>updating</b></i> <b>Ledger Apps</b>, refer to the following link:
+
+<livewire:embed-link url="https://support.ledger.com/hc/en-us/articles/360006523674-Install-uninstall-and-update-apps" caption="Install, uninstall and update apps" />
+</details>
+{{--  --}}
+
+<x-alert type="warning">
+Users should also check that they are properly **connected** to a **valid network peer**.
+</x-alert>
+
+<x-alert type="info">
+Advice on troubleshooting a **network connection** can be found <u>[here](#resolving-connectivity-issues)</u>.
+</x-alert>
+
+---
+
+### My Ledger issue is still not resolved!
+
+Most **Ledger** **connectivity** and **transaction** issues will be resolved by following the advice above.
+
+In rare cases, a users profile may have become corrupted.
+
+A user is then advised to **restore** the ARK Desktop Wallet to its **default settings**, then **re-import** wallets and watch-addresses.
+
+<x-alert type="danger">
+This will **erase a users** **profile data**, make sure <u>**all**</u> **mnemonic recovery phrases** and **watch-addresses** are **written down** and **saved properly**. Funds **will** be **lost** if your data is **not** backed up. <u>**This action cannot be undone**</u>.
```

Comment:
```suggestion
This will **erase a user's** **profile data**, make sure <u>**all**</u> **mnemonic recovery phrases** and **watch-addresses** are **written down** and **saved properly**. Funds **will** be **lost** if your data is **not** backed up. <u>**This action cannot be undone**</u>.
```